### PR TITLE
Fix/tg 237 doc meta

### PR DESCRIPTION
**Pull request by jwierzbo**
https://github.com/arangodb/go-driver/issues/387

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -29,16 +29,7 @@ language: go
 
 env:
   - TEST_SUITE=run-unit-tests GOIMAGE=gcr.io/gcr-for-testing/golang:1.16.6-stretch ALWAYS=1
-  - TEST_SUITE=run-tests-single GOIMAGE=gcr.io/gcr-for-testing/golang:1.16.6-stretch STARTER=gcr.io/gcr-for-testing/arangodb/arangodb-starter:latest ALPINE_IMAGE=gcr.io/gcr-for-testing/alpine:3.4 ARANGODB=eu.gcr.io/arangodb-ci/official/arangodb/arangodb:3.6.16
```
Comment (nikita-vanyasin) [inline .travis.yml:32]: Not sure why this test suites were removed and how it is related to the PR?

**Review by nikita-vanyasin — COMMENTED** [review]
one comment

File: .travis.yml (hunk 1 above)
Comment (jwierzbo) [reply .travis.yml:32]: This is temporary to test uknown fields. I will revert it once Travis job will pass

**Review by nikita-vanyasin — APPROVED** [review]
LGTM

**Pull request merged**